### PR TITLE
chore: add dotenv to examples 

### PR DESCRIPTION
**Pull request by haochizzle**
- add dotenv as dependency
- add `dotenv` module and .env.sample to all examples
- update code to use environment variable in replacement of embedded private key + mnemonic seed phrase

### CODE REVIEW

File: examples/evm-to-evm-fungible-transfer/src/transfer.ts [hunk 1]
```diff
@@ -1,5 +1,14 @@
 import { EVMAssetTransfer, Environment } from "@buildwithsygma/sygma-sdk-core";
 import { Wallet, providers } from "ethers";
+import * as dotenv from "dotenv";
```
Comment (wainola) [inline examples/evm-to-evm-fungible-transfer/src/transfer.ts:3]: you can do `import dotenv from 'dotenv'`

File: examples/evm-to-substrate-fungible-transfer/src/transfer.ts [hunk 2]
```diff
@@ -1,5 +1,14 @@
 import { EVMAssetTransfer, Environment } from "@buildwithsygma/sygma-sdk-core";
 import { Wallet, providers } from "ethers";
+import * as dotenv from "dotenv";
```
Comment (wainola) [inline examples/evm-to-substrate-fungible-transfer/src/transfer.ts:3]: same as above

File: examples/evm-to-evm-fungible-transfer/.gitignore
Comment (wainola) [inline examples/evm-to-evm-fungible-transfer/.gitignore:1]: this is empty

File: examples/substrate-to-evm-fungible-transfer/src/transfer.ts [hunk 3]
```diff
@@ -1,27 +1,33 @@
 import { Keyring } from "@polkadot/keyring";
 import { ApiPromise, WsProvider } from "@polkadot/api";
 import { cryptoWaitReady } from "@polkadot/util-crypto";
+import * as dotenv from "dotenv";
```
Comment (wainola) [inline examples/substrate-to-evm-fungible-transfer/src/transfer.ts:4]: same as above

**Review by wainola — CHANGES_REQUESTED** [review]

**Comment by wainola** [comment]
@haochizzle dotenv should be listed as a `devDependency`

File: examples/substrate-to-evm-fungible-transfer/src/transfer.ts (hunk 3 above)
Comment (haochizzle) [reply examples/substrate-to-evm-fungible-transfer/src/transfer.ts:4]: fixed

File: examples/evm-to-substrate-fungible-transfer/src/transfer.ts (hunk 2 above)
Comment (haochizzle) [reply examples/evm-to-substrate-fungible-transfer/src/transfer.ts:3]: fixed

File: examples/evm-to-evm-fungible-transfer/src/transfer.ts (hunk 1 above)
Comment (haochizzle) [reply examples/evm-to-evm-fungible-transfer/src/transfer.ts:3]: fixed

File: examples/evm-to-evm-fungible-transfer/.gitignore
Comment (haochizzle) [reply examples/evm-to-evm-fungible-transfer/.gitignore:1]: strange... fixed!

**Comment by haochizzle** [comment]
> @haochizzle dotenv should be listed as a `devDependencies`

updated

File: examples/evm-to-evm-fungible-transfer/src/transfer.ts [hunk 4]
```diff
@@ -1,5 +1,14 @@
 import { EVMAssetTransfer, Environment } from "@buildwithsygma/sygma-sdk-core";
 import { Wallet, providers } from "ethers";
+import dotenv from "dotenv";
+
+dotenv.config()
+
+const privateKey = process.env.PRIVATE_KEY;
+
+if (!privateKey) {
+  throw new Error("Missing environment variable");
```
Comment (MakMuftic) [inline examples/evm-to-evm-fungible-transfer/src/transfer.ts:10]:
```suggestion
  throw new Error("Missing environment variable: PRIVATE_KEY");
```

File: examples/evm-to-substrate-fungible-transfer/src/transfer.ts [hunk 5]
```diff
@@ -1,5 +1,14 @@
 import { EVMAssetTransfer, Environment } from "@buildwithsygma/sygma-sdk-core";
 import { Wallet, providers } from "ethers";
+import dotenv from "dotenv";
+
+dotenv.config()
+
+const privateKey = process.env.PRIVATE_KEY;
+
+if (!privateKey) {
+  throw new Error("Missing environment variable");
```
Comment (MakMuftic) [inline examples/evm-to-substrate-fungible-transfer/src/transfer.ts:10]:
```suggestion
  throw new Error("Missing environment variable: PRIVATE_KEY");
```

File: examples/substrate-to-evm-fungible-transfer/src/transfer.ts [hunk 6]
```diff
@@ -1,27 +1,33 @@
...
 const GOERLI_CHAIN_ID = 5;
 const RESOURCE_ID =
   "0x0000000000000000000000000000000000000000000000000000000000001000";
-const MNEMONIC =
-  "zoo slim stable violin scorpion enrich cancel bar shrug warm proof chimney";
+  const MNEMONIC = process.env.PRIVATE_MNEMONIC;
 const recipient = "0xD31E89feccCf6f2DE10EaC92ADffF48D802b695C";
 
+if (!MNEMONIC) {
+  throw new Error("Missing environment variable");
```
Comment (MakMuftic) [inline examples/substrate-to-evm-fungible-transfer/src/transfer.ts:20]:
```suggestion
  throw new Error("Missing environment variable: PRIVATE_MNEMONIC");
```

**Review by MakMuftic — APPROVED** [review]
Few minor suggestions but lgtm :+1:

File: examples/evm-to-evm-fungible-transfer/src/transfer.ts (hunk 4 above)
Comment (haochizzle) [reply examples/evm-to-evm-fungible-transfer/src/transfer.ts]: updated

File: examples/evm-to-substrate-fungible-transfer/src/transfer.ts (hunk 5 above)
Comment (haochizzle) [reply examples/evm-to-substrate-fungible-transfer/src/transfer.ts]: updated

File: examples/substrate-to-evm-fungible-transfer/src/transfer.ts (hunk 6 above)
Comment (haochizzle) [reply examples/substrate-to-evm-fungible-transfer/src/transfer.ts]: updated

**Review by itsbobbyzzz168 — APPROVED** [review]

**Pull request merged**